_posts/2021-12-14-log4j-buildpack-updates.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+date: December 14, 2021
+title: "Buildpack Updates to Mitigate log4shell exploit" 
+excerpt: Critical new updates have been released for the Java and PHP buildpacks and customers should restage their apps immediately.
+---
+
+In response to the [recently announced exploit targeting vulnerable versions of the log4j logging utility](https://github.com/advisories/GHSA-jfh8-c2jp-5v3q), the Cloud Foundry community today released patched versions of both the Java and PHP buildpacks. Customers are advised to immediately restage their applications to pick up these changes, which mitigate this new exploit. The new buildpack versions are:
```

Comment:
https://www.cisa.gov/uscert/apache-log4j-vulnerability-guidance

Let's link to this